src/Microsoft.DotNet.XHarness.CLI/CommandArguments/TestCommandArguments.cs

```diff
@@ -45,6 +47,12 @@ protected override OptionSet GetCommandOptions()
                     "ignored. Can be used more than once.",
                     v => _classMethodFilters.Add(v)
                 },
+                { "web-server-use-https", "Bind HTTPS port too",
```

Comment:
Can you please move the WASM only related args to a WASM command arguments?

`TestCommandArguments` is used across iOS and Android at the moment but the switches don't do anything there.

Applies for even the previously added ones

Reply:
We are planning to bring the same functionality to Android soon with @mdh1418

Reply:
Ah, ok. Then leave it as-is. We will soon be refactoring a bit how we define arguments so taht it's easy to assemble the sets (#431)